conversation/ui
# TC-415 Add Card docs

This PR adds documentation for the `<Card>` component.

## How to test

* Run `make storybook`
* Open `localhost:9001` and check it out

![screenshot_2018-12-10 storybook](https://user-images.githubusercontent.com/3614/49705650-fcf2a880-fc73-11e8-8c1b-9d9db95960f1.png)
